demo/src/screens/componentScreens/SkeletonViewScreen.tsx

```diff
@@ -317,6 +318,39 @@ export default class SkeletonViewScreen extends Component {
     );
   };
 
+  renderCustom = () => {
+    const {isDataAvailable} = this.state;
+
+    return (
+      <View margin-20>
+        <View row spread marginB-8>
+          <SkeletonView
+            width={120}
+            height={20}
+            borderRadius={5}
+            showContent={isDataAvailable}
+            renderContent={() => <Text text60BO>Lorem Ipsum</Text>}
+          />
+          <SkeletonView
+            width={40}
+            height={10}
+            borderRadius={5}
+            style={{marginTop: 5}}
+            showContent={isDataAvailable}
+            renderContent={() => <Button label='Info' size={'small'} link/>}
+          />
+        </View>
+        <SkeletonView
+          shimmerStyle={{width: '100%'/* , height: 30 */}}
+          height={10}
+          borderRadius={5}
+          showContent={isDataAvailable}
+          renderContent={() => <Text>Lorem Ipsum is simply dummy text of the industry.</Text>}
```

Comment:
Adding `colors={['red', 'green', 'yellow']}` to the three `SkeletonView`s will show more clearly what I meant when I wrote it's not working properly.